### PR TITLE
Update api for cpp20

### PR DESCRIPTION
**Pull request by Jasdriel**
Added c++20 standard, Is there other parts in the API that needs to be updated?

### CODE REVIEW

File: api.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ export interface SourceFileConfiguration {
     /**
      * The C or C++ standard to emulate.
      */
-    readonly standard: "c89" | "c99" | "c11" | "c++98" | "c++03" | "c++11" | "c++14" | "c++17";
+    readonly standard: "c89" | "c99" | "c11" | "c++98" | "c++03" | "c++11" | "c++14" | "c++17" | "c++20";
```
Comment (bobbrow) [inline api.ts]: Are we planning on adding a `latest` value as well?

File: api.ts [hunk 2]
```diff
@@ -137,7 +137,7 @@ export interface SourceFileConfiguration {
     /**
```
Comment (bobbrow) [inline api.ts]: We also need to update the version number as part of this change.  I can help you with this.

File: api.ts (hunk 1 above)
Comment (sean-mcmanus) [reply api.ts]: No

File: api.ts (hunk 2 above)
Comment (Jasdriel) [reply api.ts]: Ok, yes thank you

**Pull request merged**